src/main/java/frc/utils/applicationsutils/CMDHandler.java

```diff
@@ -43,7 +45,8 @@ public static void runJavaClass(Path javaPath) {
     public static void runJavaClass(Path javaPath, String arguments) {
```

Comment:
```suggestion
    public static void runJavaClass(Path javaPath, String... arguments) {
```

Reply:
each argument is sperated by a space, so getting multiple is unnecessary

Reply:
I don’t want you to get them as string each one separately